frontends/qt/main.cpp

```diff
@@ -162,10 +175,38 @@ int main(int argc, char *argv[])
     a.setOrganizationDomain("shiftcrypto.ch");
     a.setOrganizationName("Shift Crypto");
```

Comment:
You could add a comment here saying that, if changed, it may break "single app" mode: app name and org are all part of the qlocalserver name. https://github.com/digitalbitbox/SingleApplication/blob/c557da5/singleapplication_p.cpp#L135-L137

Reply:
Done.

Speaking of, maybe we should also add a message if version changed, so the user knows to close the running app to launch the new one - or they will simply bring the old one to the foreground. Maybe in another PR.